golang/src/com/google/idea/blaze/golang/util/FileUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.google.idea.blaze.golang.util;
+
+import java.io.File;
+
+public class FileUtil {
```

Comment:
If we end up using patchExternalFilePath() in only one location, please convert it to a private method.

If we need to keep this extra class, we 
1) need a private constructor so that the class can never be instantiated and
2) need Javadoc descriptions on the class and public method.